doc/requirements.txt

```diff
@@ -2,4 +2,5 @@ sphinx >=4.0.0
 sphinx-fortran
 sphinx_rtd_theme
 sphinxcontrib.bibtex
-sphinx-copybutton
\ No newline at end of file
+sphinx-copybutton
+graphviz
```

Comment:
The `graphviz` python package isn't necessary to build the docs. There needs to be a [Graphviz installation](https://graphviz.org/download/) on the system (this is also required by the python package). This requirement is noted in `doc/README.md`